doc/nrf/libraries/bluetooth_services/services/hids.rst

```diff
@@ -7,14 +7,14 @@ GATT Human Interface Device (HID) Service
    :local:
    :depth: 2
 
-This module implements the Human Interface Device Service with the corresponding set of characteristics.
+This library implements the Human Interface Device Service (HIDS) with the corresponding set of characteristics.
```

Comment:
btw. This one could be called a service.
I actually wonder if we should always explicitly use `service` term for `GATT Services` and `library` term for other use-cases? (e.g. Fast Pair GATT Service doc already uses `service` term).

If we opt for rename, please also align other similar occurrences in the PR.

Reply:
Would be a bit odd to say "This service implements the ... service."

Reply:
Ok, then let's leave it as is to avoid bigger rephrasing.